src/main/java/up/value/chefleaserver/service/PopupService.java

```diff
@@ -19,19 +20,24 @@ public class PopupService {
 
     private final PopupRepository popupRepository;
 
+    private final UserService userService;
+
     @Transactional(readOnly = true)
     public PopupsGetResponse getAllPopups(User loginUser) {
         List<Popup> popups = popupRepository.findAll();
 
         List<PopupGetResponse> popupGetResponses = popups.stream()
                 .map(popup -> {
-                    boolean isLiked = popup.getPopupLikes().stream().map(PopupLike::getUser)
-                            .toList()
-                            .contains(loginUser);
+                    boolean isLiked = userService.isLikedByUser(loginUser, popup);
                     return PopupGetResponse.of(popup, isLiked);
                 })
                 .toList();
 
         return PopupsGetResponse.of(popupGetResponses);
     }
+
+    public PopupDetailGetResponse getPopup(User loginUser, Long popupId) {
+        Popup popup = popupRepository.findById(popupId).orElseThrow(RuntimeException::new);
```

Comment:
개행하면 가독성이 더 좋아질 것 같아요
```suggestion
        Popup popup = popupRepository.findById(popupId)
                .orElseThrow(RuntimeException::new);
```